### DISCUSSION TITLE
Thanks for making this fork

### DISCUSSION THREAD

**Discussion by gitdrug** [tenacityteam/tenacity-legacy#278] [General]
thanks for making this fork so i dont have bad spywere

**Comment by poyribn** [comment]
I also appreciate this fork like this.

For now I'd personally still use audacity
but within a Sandbox
(like with the Linux command `firejail --net=none audacity`)
because compiling Tenacity (or any software) myself
seems way too complicated for me.

But as soon as the first release is ready,
i will Immediately switch to Tenacity.

**Comment by Songtech-0912** [comment]
All in favor! Enough said, Tenacity is awesome!
